.travis.yml

```diff
@@ -36,12 +36,12 @@ licenses:
     - 'android-sdk-license-.+'
     - 'google-gdk-license-.+'
 
-script: 
+script:
  - ./gradlew build
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
   - 'if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./upload-apk.sh; fi'
 
 notifications:
-  slack: fossasia:JgzycrBUs0nKnmJhsAxCB4bL
+  slack: fossasia:JgzycrBUs0nKnmJhsAxCB4bL
```

Comment:
These are unnecessary changes. Please configure you editor according to the project conventions.